CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.25)
+project(libreadtags)
+
+set(CMAKE_C_STANDARD 11)
```

Comment:
Unlike ctags itself, I would like to keep libreadtags compilable with C99.

Reply:
I've just found `AC-PROG-CC-C99` in configure.ac. I'll update that.